### DISCUSSION TITLE
Markdown slot is no longer supported. Now what ?

### DISCUSSION THREAD

**Discussion by sylvainpolletvillard** [vuepress/core#1440] [Q&A] [answered]
I'm migrating a vuepress website to v2 and just noticed that markdown slots were no longer supported.

I use markdown slot for a basic component which acts as a "spoiler" tag to show some markdown only when clicking on it:

```
<Solution>
```json
{
answer: 42
}
\```
</Solution>
```

If markdown slots are no longer supported, I need to find another way to do the same thing with vuepress 2

I tried rendering manually markdown-it on the slot content but there is a change in Vue 3 with slots which make it extremely difficult to get the slot content as text and with all whitespaces/linebreaks preserve.

What's the migration plan here ? Can you expose some internals of vuepress so we could plug the markdown renderer in our components ?

Thanks

**Comment by Mister-Hope** [comment, marked as answer]
it seems that a markdown-it plugin is what you want, check component option in vuepress-plugin-md-enhance
